src/main/java/com/prototype/booking/api/referencedata/Timings.java

```diff
@@ -0,0 +1,29 @@
+package com.prototype.booking.api.referencedata;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import lombok.Data;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "TIMINGS")
+@Data
+public class Timings {
+
+    @Id
+    @Column
+    private String timeId;
+
+    @Column
+    private String timeSlot;
+
+    @JsonIgnore
+    @Column
+    private String roomCode;
```

Comment:
```suggestion
    private String roomId;
```

Reply:
Yes after primary key column change in MeetingRoom entity as id, roomCode now can change to roomId in Timings entity as a foreign key